### PR TITLE
consul tag support

### LINKED ISSUE #109: Allow Consul tags for observability / filter purposes
## Feature description
Add input variable to module with list of tags to declare in consul.

## Why is it needed?
Discoverability and observability

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### PR DESCRIPTION
**Pull request by dangernil**
## Closes/fixes/resolves issue(s)?
Closes #109 

## What was added/changed/fixed?
- Consul tags support via input variable

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -15,6 +15,13 @@ variable "nomad_host_volume" {
   default     = "persistence"
 }
 
+variable "consul_tags" {
+  type        = list(string)
+  description = "List of tags for services announces in consul service catalog"
+  default     = [" "]
```
Comment (claesgill) [inline variables.tf]: Think default should be empyt or prehaps `"minio"`. WDYT?

File: main.tf [hunk 2]
```diff
@@ -75,8 +76,8 @@ data "template_file" "nomad_job_mc" {
 }
 
 resource "nomad_job" "nomad_job_minio" {
-  jobspec = data.template_file.nomad_job_minio.rendered
-  detach  = false
+  jobspec                     = data.template_file.nomad_job_minio.rendered
+  detach                      = false
```
Comment (claesgill) [inline main.tf:80]:
```suggestion
  jobspec = data.template_file.nomad_job_minio.rendered
  detach  = false
```

File: main.tf [hunk 3]
```diff
@@ -33,6 +33,7 @@ data "template_file" "nomad_job_minio" {
     host_volume               = var.nomad_host_volume
     image                     = var.container_image
     service_name              = var.service_name
+    consul_tags               = join(",", var.consul_tags)
```
Comment (claesgill) [inline main.tf:36]: Can you update README with this new variable?

File: variables.tf (hunk 1 above)
Comment (dangernil) [reply variables.tf]: I´ll leave it empty

File: main.tf (hunk 3 above)
Comment (dangernil) [reply main.tf:36]: Yes, thanks for reminding me

File: README.md [hunk 4]
```diff
@@ -121,6 +121,7 @@ module "minio" {
 | nomad\_namespace | [Enterprise] Nomad namespace | string | "default" | yes |
 | nomad\_host\_volume | Nomad host volume | string | "persistence" | no |
 | service\_name | Minio service name | string | "minio" | yes |
+| consul\_tags | Consul tags | list(string) | "" | no |
```
Comment (zhenik) [inline README.md]:
```suggestion
| consul\_tags | Consul tags | list(string) | [""] | no |
```

**Review by zhenik — CHANGES_REQUESTED** [review]

**Review by zhenik — APPROVED** [review]

**Review by claesgill — APPROVED** [review]

**Pull request merged**